packages/storybook/package.json

```diff
@@ -13,22 +14,26 @@
   },
   "devDependencies": {
     "@babel/core": "^7.15.8",
+    "@babel/preset-env": "^7.24.0",
+    "@babel/preset-react": "^7.23.3",
+    "@babel/preset-typescript": "^7.23.3",
     "@department-of-veterans-affairs/formation": "^11.0.5",
     "@mdx-js/react": "^1.6.22",
-    "@storybook/addon-a11y": "6.4.19",
-    "@storybook/addon-actions": "^6.4.19",
-    "@storybook/addon-docs": "^6.4.19",
-    "@storybook/addon-essentials": "^6.4.19",
-    "@storybook/addon-links": "^6.4.19",
-    "@storybook/addon-storysource": "^6.4.19",
-    "@storybook/builder-webpack5": "^6.4.19",
-    "@storybook/manager-webpack5": "^6.4.19",
-    "@storybook/react": "^6.4.19",
-    "@whitespace/storybook-addon-html": "^5.0.0",
+    "@storybook/addon-a11y": "7.6.19",
+    "@storybook/addon-actions": "^7.6.10",
+    "@storybook/addon-docs": "^7.6.10",
+    "@storybook/addon-essentials": "^7.6.10",
+    "@storybook/addon-links": "^7.6.10",
+    "@storybook/addon-mdx-gfm": "7.6.19",
+    "@storybook/addon-storysource": "^7.6.10",
+    "@storybook/react": "^7.6.10",
+    "@storybook/react-webpack5": "7.6.19",
     "babel-loader": "^8.2.2",
+    "chromatic": "^11.5.1",
     "css-loader": "^5.2.4",
-    "node-sass": "^7.0.0",
+    "node-sass": "npm:sass@^1.49.9",
```

Comment:
I did this because my local build was failing for the `web-components` package during debugging. Is it still needed?

Reply:
@jamigibbs switched back and everything still works